### PR TITLE
Style footer

### PR DESCRIPTION
**Pull request by shreyshreyansh**
aaaa

### CODE REVIEW

**Review by shreyshreyansh — COMMENTED** [review]
Changed the file

**Review by nehakalbande — COMMENTED** [review]
Can you please share mobile version ss!

**Comment by shreyshreyansh** [comment]
sas

**Comment by shikha-16** [comment]
> sas

Hi Shreyansh, this is a nice footer! However, please edit the about section to the following -

Women in Technology empowers women and young girls wanting to pursue Computer Science all around the world by increasing visibility of resources and opportunities, sharing inspirational stories of successful professionals in STEM and through our amazing Discord community of women uplifting each other.

**Comment by shreyshreyansh** [comment]
**Also, what is in Privacy Policy and the Copyright section?**

I just added those links, just in case in the future you wanted to edit those

File: website/src/components/Footer/Footer.js [hunk 1]
```diff
@@ -1,75 +1,89 @@
...
+                <a href="#">About Us</a>
+              </li>
+              <li>
+                <a href="#">Contact Us</a>
+              </li>
+              <li>
+                <a href="#">Contribute</a>
+              </li>
+              <li>
+                <a href="#">Privacy Policy</a>
```
Comment (shreyshreyansh) [inline website/src/components/Footer/Footer.js:41]: I have kept those links blank in case you want to edit those

**Comment by nehakalbande** [comment]
Hey @shreyshreyansh please do this corrections:
1. add footer bgcolor #ffcc66 
2. about section : Women in Technology empowers women and young girls wanting to pursue Computer Science all around the world by increasing visibility of resources and opportunities, sharing inspirational stories of successful professionals in STEM and through our amazing Discord community of women uplifting each other.
3. text color to be in black.

**Pull request closed without merging**